api/files/serializers.py

```diff
@@ -5,8 +5,9 @@
 
 from modularodm import Q
 
-from website import settings
 from framework.auth.core import Auth, User
+from website import settings
```

Comment:
This, incidentally, is why I discourage re-shuffling imports that don't actually affect functionality.